eth/transaction.go

```diff
@@ -25,7 +25,7 @@ type Transaction struct {
 	From        Address   `json:"from"`
 	Gas         Quantity  `json:"gas"`
 	Hash        Hash      `json:"hash"`
-	Input       Data      `json:"input"`
+	Input       Input     `json:"input"`
```

Comment:
looks like `eth.Transaction` is used in several other repos as well. not sure how many would need to be updated after changing this type.

Reply:
🙏 good catch